packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -22,8 +22,8 @@ export interface RollupLwcOptions {
     exclude?: FilterPattern;
     /** The LWC root module directory. */
     rootDir?: string;
-    /** If `true` the plugin will produce source maps. */
-    sourcemap?: boolean;
+    /** If `true` the plugin will produce source maps. If `'inline'`, the plugin will produce inlined source maps and append them to the end of the generated file. */
+    sourcemap?: boolean | 'inline';
```

Comment:
This might actually need to be passed to rollup and not to the LWC compiler to give to Babel; Rollup will throw away the inline sourcemaps. This is also true of non-inline sourcemaps, so it's possible this has never worked and no one's brought it up.

Setting this property in our perf-benchmarks-components project does nothing to the output even though LWC is properly passing the value down to the Babel transform.

Reply:
Generally, no sourcemaps will be generated if rollup itself isn't set to generate sourcemaps; it'll delete any existing inline sourcemaps it finds. If Rollup is set up to generate sourcemaps, there are [other plugins](https://www.npmjs.com/package/rollup-plugin-sourcemaps) that can help it load files with existing sourcemaps. This seems like a very small edge case for us though. If the sourcemaps generated by rollup are correct (need to verify), we should be fine relying on it instead of Babel.